pkg/koordlet/util/system/resource.go

```diff
@@ -57,15 +56,15 @@ func ValidateResourceValue(value *int64, parentDir string, r Resource) bool {
 	return true
 }
 
-func SupportedIfFileExists(r Resource, parentDir string) (*bool, string) {
+func SupportedIfFileExists(r Resource, parentDir string) (*bool, bool, string) {
 	exists, err := PathExists(r.Path(parentDir))
 	if err != nil {
-		return pointer.Bool(false), fmt.Sprintf("cannot check if %s exists, err: %v", r.ResourceType(), err)
+		return nil, false, fmt.Sprintf("cannot check if %s exists, err: %v", r.ResourceType(), err)
 	}
 	if !exists {
-		return pointer.Bool(false), "file not exist"
+		return nil, false, "file not exist"
 	}
-	return pointer.Bool(true), ""
+	return nil, true, ""
```

Comment:
seems the new returned boolean is not necessary

Reply:
removed